core/src/test/java/com/yahoo/gondola/rc/RcChannel.java

```diff
@@ -197,7 +198,9 @@ public void write(byte[] b, int off, int len) throws IOException {
                 }
             } catch (Exception e) {
                 message.release();
-                throw new IOException("key=" + key, e);
+                if (!"Read end dead".equals(e.getMessage())) {
```

Comment:
if possible, wrap this to a new exception class will be better if there is a change for the message in the bottom layer.